packages/ui/src/Field.tsx

```diff
@@ -132,7 +140,9 @@ export const Field = ({
           name={name}
           switched={Boolean(value)}
           testID={testID}
-          onChange={(result) => handleSwitchChange(result)}
+          onChange={(result) => {
+            handleSwitchChange(result);
```

Comment:
Do we want to onBlur here too? It's kinda a weird line to have select field do it but not switch, but I could go any of the 3 ways (both, none, only Select)

Reply:
This actually does call `onBlur` already if you look at the `handleSwitchChange` function.

Reply:
Ah duh!